sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelHandleTest.java

```diff
@@ -83,6 +83,8 @@ void setUp() {
     @TestIdentifier(EnabledTestConfig.BICEPS_R0007_0)
     @TestDescription("Starting from the initially retrieved mdib, ensures that for each mdib version, "
             + " all contained handles are unique.")
+    @RequirePrecondition(
+            manipulationPreconditions = {ManipulationPreconditions.RemoveAndReinsertDescriptorManipulation.class})
```

Comment:
Since the insertion manipulation is handle specific, I doubt that it gives any benefit here, since a correct implementation of the insertion manipulation is supposed to adhere to the handle parameter given.